cmd/rcd/rcd.go

```diff
@@ -32,18 +41,209 @@ See the [rc documentation](/rc/) for more info on the rc flags.
 		if rcflags.Opt.Enabled {
 			log.Fatalf("Don't supply --rc flag when using rcd")
 		}
+
 		// Start the rc
 		rcflags.Opt.Enabled = true
 		if len(args) > 0 {
 			rcflags.Opt.Files = args[0]
 		}
+
+		if rcflags.Opt.WebUI {
+
```

Comment:
I think this would be better factored out into a helper functiion `checkRelease` or something like that.